cluster/manifests/kube-state-metrics/deployment.yaml

```diff
@@ -24,17 +24,32 @@ spec:
       priorityClassName: system-cluster-critical
       serviceAccountName: kube-state-metrics
       containers:
-      - image: registry.opensource.zalan.do/teapot/kube-state-metrics:v1.3.0
-        name: kube-state-metrics
+      - name: kube-state-metrics
+        image: registry.opensource.zalan.do/teapot/kube-state-metrics:v1.8.0
         ports:
         - containerPort: 8080
-        command:
-        - /kube-state-metrics
-        - --port=8080
+          name: http-metrics
+        - containerPort: 8081
+          name: telemetry
+        readinessProbe:
+          httpGet:
+            path: /
+            port: 8081
+          initialDelaySeconds: 5
+          timeoutSeconds: 5
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8080
+          initialDelaySeconds: 5
+          timeoutSeconds: 5
```

Comment:
Why same configuration as readinessprobe?

Reply:
It's using different endpoints (metrics port vs. telemetry port). So it might make sense even with the same interval configuration. It's from [their repo](https://github.com/kubernetes/kube-state-metrics/blob/master/examples/standard/deployment.yaml#L22-L27). I haven't checked what the telemetry port exposes.

Reply:
might makes sense, thanks for the hints

Reply:
Actually there's not much difference. One serves the "kubernetes metrics" (number of pods etc.) the other one is metrics of `kube-state-metrics` itself (goroutines).

If we want to follow the "never use the same timings for readiness and liveness" approach I'm happy to change it.

Reply:
Resolved by dropping the `liveness` probe.